src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -441,7 +444,11 @@ class SpanDetail extends React.Component<Props, State> {
               <Tags span={span} />
               {map(span?.data ?? {}, (value, key) => (
                 <Row title={key} key={key}>
-                  {JSON.stringify(value, null, 4) || ''}
+                  {SIZE_DATA.has((key as unknown) as string) ? (
+                    <FileSize bytes={(value as unknown) as number} />
+                  ) : (
+                    JSON.stringify(value, null, 4) || ''
+                  )}
```

Comment:
Can we add the actual values in parenthesis? e.g. `3.4 KB (3401)`

Reply:
@k-fish suggested using a tooltip for that. Would that work for you? We have that pattern in other places as well.

Reply:
It might not be easy to add a tooltip here from the last time I worked with this component. 

Have you tried to attempt to add it? 

Text only might be the simplest.